CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## [Next]
+ - Refactored the particle system
```

Comment:
Is this not part of another PR?

Reply:
I forgot all the changelog entries for the massive PR I just merged and I was lazy and just added them to this one instead 😄